docs/manual/java/guide/production/ProductionOverview.md

```diff
@@ -1,10 +1,6 @@
 # Production
 
-Lagom doesn't prescribe any particular production environment, however out of the box support is provided for [Lightbend ConductR](https://www.lightbend.com/products/conductr).
-
-## Deploying Lagom services to ConductR
-
-Lightbend ConductR is a perfect match for Lagom, as it provides the following features:
+Lagom doesn't prescribe any particular production environment. If you are interested in deploying on Kubernetes, see our guide that demonstrates [how to deploy the Chirper example application](https://developer.lightbend.com/guides/k8s-microservices/). Out-of-the-box support is provided for [Lightbend ConductR](https://www.lightbend.com/products/conductr). Lightbend ConductR is a perfect match for Lagom, as it provides the following features:
```

Comment:
I'm surprised k8s is listed first. I'd keep conductr as first choice and after it's benefits (list below) I'd mention k8s.